templates/main/group_builds.html.ep

```diff
@@ -1,55 +1,57 @@
-% for my $build_res (@$build_results) {
-    % my $build = $build_res->{build};
-    % my $group_id;
-    <div class="row build-row <%= $children ? ($default_expanded ? 'children-expanded' : 'children-collapsed') : 'no-children' %>">
-        <div class="col-lg-4 text-nowrap">
-            <h4>
-                % if ($children) {
-                    % $group_id = "group$group->{id}_build$build_res->{escaped_id}";
-                    <a href="#<%= $group_id %>" onclick="return toggleChildGroups(this);" data-toggle="collapse">Build<%= $build %></a>
-                % } else {
-                    %= link_to "Build$build" => url_for('tests_overview')->query(distri => $build_res->{distri}, version => $build_res->{version}, build => $build, groupid => $group->{id} )
-                % }
+<div id="build-results">
```

Comment:
Yes, it can be used as an anchor.

BTW: The changed lines above (except closing `</div>`) only have different indentation but no further adjustments.